infra/reachability-metadata/src/main/resources/META-INF/native-image/org.apache.shardingsphere/generated-reachability-metadata/resource-config.json

```diff
@@ -0,0 +1,2822 @@
+{
+  "resources":{
+  "includes":[{
+    "condition":{"typeReachable":"org.apache.shardingsphere.infra.expr.groovy.GroovyInlineExpressionParser"},
+    "pattern":"\\QMETA-INF/dgminfo\\E"
+  }, {
```

Comment:
- In fact, just adjusting the User CodeFilter and Extra Filter will not restrict `resource-config.json` from outputting the specified entries. The contents of this file include test classes and third-party dependency files such as Groovy that we do not want to include. This is a limitation of GraalVM CE's `native-image-configure`.